utils/emmylua/README.md

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
???

Reply:
The first line of the file is blank, the text doesn't start until line 2.

Reply:
???

Reply:
Because it seems strange; is there meant to be a blank line at the start of the file?

Reply:
I don't think it's all that strange. I don't really remember, but I probably did that on purpose.

Reply:
That said, if this is something that's meant to be standardized in Wesnoth's codebase, it should be done so as part of the whitespace checker script rather than one-off comments in a particular PR.